app/controllers/api/v1/base_controller.rb

```diff
@@ -35,6 +35,22 @@ def currencies
       render json: Dictionaries.currencies
     end
 
+    def currency_rates
+      bank = Money.default_bank
+      rates = bank.rates.inject([]) do |result, (k, v)|
```

Comment:
@dissident you should move it separate model or services or... anything. But this logic should not be in controller.